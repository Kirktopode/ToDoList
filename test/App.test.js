import React from 'react'
import { render } from 'react-dom'
import { Provider } from 'react-redux'
import { createStore } from 'redux'
import todoApp from '../src/reducers'
import App from '../src/components/App'
import renderer from 'react-test-renderer';

test('App renders without crashing.', () =>{

  let store = createStore(todoApp)

  const component = renderer.create(
    <Provider store={store}>
      <App />
    </Provider>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
