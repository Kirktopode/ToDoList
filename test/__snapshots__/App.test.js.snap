// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing. 1`] = `
<div>
  <div>
    <form
      onSubmit={[Function]}
    >
      <input />
      <button
        type="submit"
      >
        Add Todo
      </button>
    </form>
  </div>
  <ul />
  <p>
    Show:
     
    <span>
      All
    </span>
    , 
    <a
      href=""
      onClick={[Function]}
    >
      Active
    </a>
    , 
    <a
      href=""
      onClick={[Function]}
    >
      Completed
    </a>
    <input
      checked={true}
      onChange={[Function]}
      type="checkbox"
    />
    Time Added
  </p>
</div>
`;
